.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Local logging
+log/
```

Comment:
Ok, I was wrong to suggest this, the file handler can't be configured without the file there. Get rid of this.

Make a `log` directory and include an empty text file called `.gitkeep` inside it so that it is included.